front/app/hooks/useAdminPublicationsStatusCounts.ts

```diff
@@ -42,13 +43,18 @@ export default function useAdminPublicationsStatusCounts({
     areas.length === 0 ? setAreas(undefined) : setAreas(areas);
   }, []);
 
+  const onChangeSearch = useCallback((search: string) => {
+    search && search.length === 0 ? setSearch(null) : setSearch(search);
```

Comment:
search is always going to be a string (as per parameter types). No need to check `search &&`

Reply:
Good catch, it can be null as well. I've updated it

Reply:
Just saw it will never be null, so let's narrow the type again. It's only used in front/app/components/ProjectAndFolderCards/ProjectAndFolderCardsInner.tsx and can only be string there.